pkg/services/alerting/engine.go

```diff
@@ -101,20 +104,21 @@ func (e *Engine) processJob(grafanaCtx context.Context, job *Job) error {
 	}()
 
 	alertCtx, cancelFn := context.WithTimeout(context.Background(), alertTimeout)
+	span := opentracing.StartSpan("alert execution")
+	alertCtx = opentracing.ContextWithSpan(alertCtx, span)
 
 	job.Running = true
 	evalContext := NewEvalContext(alertCtx, job.Rule)
-
-	done := make(chan struct{})
-
-	span := opentracing.StartSpan("alerting")
-	alertCtx = opentracing.ContextWithSpan(alertCtx, span)
 	evalContext.Ctx = alertCtx
 
+	done := make(chan struct{})
 	go func() {
 		defer func() {
 			if err := recover(); err != nil {
 				e.log.Error("Alert Panic", "error", err, "stack", log.Stack(1))
+				ext.Error.Set(span, true)
+				span.LogFields(tlog.Error(fmt.Errorf("error: %v", err)))
```

Comment:
why the `error :` prefix? it's already clear that this is an error

Reply:
event=error message=xyz

https://github.com/opentracing/specification/blob/master/semantic_conventions.md